sharktank/sharktank/ops/signatures.py

```diff
@@ -374,6 +376,23 @@ def _gather_trampoline(
         d.fail(dispatch_args)
 
 
+def gelu_tanh_approximation(input: AnyTensor) -> AnyTensor:
```

Comment:
lots of hardcoded values here. Are those all specified by the linked paper?

Reply:
Yes, this approximation is proposed in the paper.